lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -1612,3 +1613,11 @@ func setEpcNetworkServiceAreaMap(nw *configurator.Network, serviceAreaMaps map[s
 	cellularNwConfig.Epc.ServiceAreaMaps = serviceAreaMaps
 	nw.Configs[lte.CellularNetworkConfigType] = cellularNwConfig
 }
+
+func getRandomizedInterval(nwID string, gwKey string, interval uint32) uint32 {
```

Comment:
Eh just make this public instead of copying, though idc either way

Reply:
It is weird to have the function being tested directly provide the expected value. Problems with the function wouldn't be caught. I could hardcode the expected randomized values if we don't want to copy the function...

Reply:
Agreed. Let's go with hard-coding then

Reply:
Done.